src/dlangui/core/events.d

```diff
@@ -1649,6 +1649,20 @@ class RunnableEvent : CustomEvent {
     }
 }
 
+/// queue destroy event
```

Comment:
Most of summary I added to function Window.queueWidgetDestroy() but also added links between event and Window function.